src/status_im/ui/screens/browser/views.cljs

```diff
@@ -64,9 +65,15 @@
     [components/activity-indicator {:animating true}]]))
 
 (defn on-navigation-change [event browser]
-  (let [{:strs [url canGoBack canGoForward]} (js->clj event)]
-    (when (and (not (:dapp? browser)) (not= "about:blank" url))
-      (re-frame/dispatch [:update-browser (assoc browser :url url)]))
+  (let [{:strs [url canGoBack canGoForward loading]} (js->clj event)]
+    (when (not= "about:blank" url)
+      (let [browser2 (if (not (:dapp? browser)) (assoc browser :url url) browser)]
```

Comment:
we needed this because we didn't have history, in this PR we can remove this condition `(not (:dapp? browser)) `

Reply:
Done!